trade-finance-manager-ui/scripts/enable-find-utilisation-reports-by-year-dropdown-autocomplete.ts

```diff
@@ -0,0 +1,29 @@
+const enableFindUtilisationReportsByYearDropdownAutocomplete = (): void => {
+  const findUtilisationReportsByYearForm: HTMLFormElement | null = document.querySelector('form#find-utilisation-reports-by-year--form');
+
+  if (!findUtilisationReportsByYearForm) {
+    return;
+  }
+
+  const yearInput: HTMLInputElement | null = document.querySelector('input#find-utilisation-reports-by-year--year-input');
```

Comment:
Are these inputs part of utilisation report year form, wondering if yes then can we do `findUtilisationReportsByYearForm.querySelectorAll...`

Reply:
There is only one input for the year, hence `querySelector` (not to mention that we want to differentiate between the radio inputs and this specific input)